services/app-api/src/postgres/contractAndRates/contractType.ts

```diff
@@ -0,0 +1,27 @@
+import { UpdateInfoType } from "../../domain-models"
+import { RateRevision } from "./rateType"
+
+interface Contract {
+    id: string
+    revisions: ContractRevision[]
+}
+
+// there needs to be a new contract revision after every set, even though the revision is the same?
```

Comment:
Can we please organize comments a bit more before we merge.  Right now I can't tell if these ?? question??? statements are 1. open questions that need to be answered 2. outdated dev notes 3. more philosophical wondering or marginalia that I can ignore working int his file. 

Let's try to clean this up- make it more clear what needs further action or consideration in this file and what doesn't